spec/sidekiq/mhv/audit_login_job_spec.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require 'support/rx_client_helpers'
+
+RSpec.describe MHV::AuditLoginJob do
+  let(:mhv_correlation_id) { '12345' }
+  let(:authenticated_client) do
+    MHVLogging::Client.new(session: { user_id: mhv_correlation_id })
+  end
+
+  before do
+    allow(MHVLogging::Client).to receive(:new).and_return(authenticated_client)
+    allow(authenticated_client).to receive(:authenticate).and_return(authenticated_client)
+    allow(authenticated_client).to receive(:auditlogin)
+  end
+
+  describe 'perform' do
+    it 'calls MHV audit client with correlation id' do
```

Comment:
This test and the last one with the cassette are basically testing the same thing. You could combine them